csrc/python_frontend/fusion_state.cpp

```diff
@@ -68,6 +68,13 @@ void FusionState::resetFusionState(Fusion* fusion, size_t size) {
   fusion_state_.resize(size, nullptr);
 }
 
+void FusionState::addFusionState(size_t index, Val* val) {
```

Comment:
looks like a VERY ugly API! let's not do this here.

Reply:
Changed.